Events/src/main/java/com/github/swcraftlyon/meetup/Event.java

```diff
@@ -1,4 +1,17 @@
 package com.github.swcraftlyon.meetup;
 
 public class Event {
+    private String title;
+
+    public Event(String title) {
+        this.title = title;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
```

Comment:
Actually, there probably is no need for a setter in this case.  Setters mean that an object is mutable, and this may lead to complexity in the future or even inconsistent state in a multithread environment.

Remove the setter and put a final keyword on the field and that should do.